### PR TITLE
Type definitions for parity-pmd 0.4.0

### PR DESCRIPTION
**Pull request by vilimco**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/parity-pmd/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Type definitions for parity-pmd 0.4
+// Project: https://github.com/paritytrading/node-parity-pmd#readme
+// Definitions by: Leonard Vujanić <https://github.com/leovujanic>
```
Comment (leovujanic) [inline types/parity-pmd/index.d.ts:3]: @jewbre can you please add your name here?
Here is a snippet
```
// Definitions by: Alice <https://github.com/alice>
//                          Bob <https://github.com/bob>
```

**Review by leovujanic — CHANGES_REQUESTED** [review]

**Comment by leovujanic** [comment]
Please add your signature and you are ready to go. :) Checked everything locally and it all seems good to me.

File: types/parity-pmd/index.d.ts (hunk 1 above)
Comment (vilimco) [reply types/parity-pmd/index.d.ts:3]: Since it was your type, I didn't even think about that. :)
Added.

**Comment by vilimco** [comment]
@leovujanic Re-review? :)

**Review by leovujanic — APPROVED** [review]
👍

**Pull request merged**